dspace/config/pages/help.xml

```diff
@@ -2,38 +2,51 @@
 
 <body>
 <div id="file.help.div.help" n="help" rend="primary">
-<head>MOspace Help</head>
-
+<head>MOspace Institutional Repository Help</head>
 
 <div id="top" n="top">
+<body>
```

Comment:
Changes made.
Felicity

From: Hardy Pottinger [mailto:notifications@github.com]
Sent: Tuesday, February 09, 2016 5:05 PM
To: umlso/DSpace
Cc: Dykas, Felicity A.
Subject: Re: [DSpace] Update help.xml (#77)


In dspace/config/pages/help.xml<https://github.com/umlso/DSpace/pull/77#discussion_r52389615>:

>

>  <div id="top" n="top">

> +<body>

the body tag does not belong here, it's a wrapper for the entire page (see line 3)

—
Reply to this email directly or view it on GitHub<https://github.com/umlso/DSpace/pull/77/files#r52389615>.